.github/workflows/keg-build.yml

```diff
@@ -0,0 +1,47 @@
+
+name: keg-build
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  tests:
+    name: 'Build Repos'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: "Get Changed Files"
+        id: get-changed-files
+        run: bash scripts/ci/getChangedFiles.sh
+
+      - name: "Get Changed Repos"
+        id: get-changed-repos
+        run: bash scripts/ci/getChangedRepos.sh
+
+      - name: "Install Yarn Dependencies"
```

Comment:
I will re-test again tomorrow. but after testing it tonight here's what I found: 
1. if the `format/test/build` steps fail. it will still count it as **success**
    - see [this](https://github.com/daniel112/keg-hub/actions/runs/464965447) workflow for when `format` step throws an error
2. jsutils  fails at yarn build  when on node 14.15.1  (default)
      - see `run repo build` step: https://github.com/daniel112/keg-hub/runs/1653846780?check_suite_focus=true
      - node v 14.15.1 is default for `ubuntu-latest` runner
      - this is the error when running `yarn build` on  node `14.15.1` - [image](https://user-images.githubusercontent.com/3317835/103721867-fbe3f080-4f8b-11eb-8cbd-90a3f105de10.png)
      - I manage to bypass this by setting the node version to `12` on the workflow (since we currently use this major version)
      - ```js
                   - name: "Setup node Version"
                      uses: actions/setup-node@v2
                      with:
                        node-version: '12.x.x'
         ```
      - see the workflow that successfully builds jsutils [here](https://github.com/daniel112/keg-hub/runs/1653917635?check_suite_focus=true)

Reply:
* Updated bash scripts to exit on error
* Updated all workflows to run on version 12 of node
 * Updated Keg-Components and Keg-Core to pass formatter script
 * ReTheme needs a number of updates so holding off on that one